src/main/java/duke/command/ListTasksCommand.java

```diff
@@ -0,0 +1,49 @@
+package duke.command;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * Represents a command that lists the user's tasks.
+ *
+ * @author wz2k
+ */
+public class ListTasksCommand extends Command {
+    /** The list of task maintained by the chatbot */
+    private TaskList taskList;
+
+    /**
+     * Creates a command for listing the user's tasks.
+     *
+     * @param commandMessage User's input.
+     * @param taskList List of tasks.
+     */
+    public ListTasksCommand(String commandMessage, TaskList taskList) {
+        super(commandMessage);
+        this.taskList = taskList;
+    }
+
+    /**
+     * Lists the tasks for the user and returns the reply with the list
+     * of found tasks
+     *
+     * @return the list of found tasks.
```

Comment:
```suggestion
     * @return The list of found tasks.
```

Reply:
Thanks for helping to spot that small error 👍